easybuild/easyconfigs/h/HapCUT2/HapCUT2-v1.3.3-GCC-12.3.0.eb

```diff
@@ -0,0 +1,75 @@
+easyblock = 'MakeCp'
+name = 'HapCUT2'
+version = 'v1.3.3'
+
+homepage = 'https://github.com/vibansal/HapCUT2/'
+description = """
+HapCUT2 is a maximum-likelihood-based tool for assembling haplotypes from DNA sequence reads,
+designed to "just work" with excellent speed and accuracy.
+We found that previously described haplotype assembly methods are specialized for specific read technologies or
+protocols,
+with slow or inaccurate performance on others.
+With this in mind, HapCUT2 is designed for speed and accuracy across diverse sequencing technologies,
+including but not limited to:
+
+    - NGS short reads (Illumina HiSeq)
+    - single-molecule long reads (PacBio and Oxford Nanopore)
+    - Linked-Reads (e.g. 10X Genomics, stLFR or TELL-seq)
+    - proximity-ligation (Hi-C) reads
+    - high-coverage sequencing (>40x coverage-per-SNP) using above technologies
+    - combinations of the above technologies (e.g. scaffold long reads with Hi-C reads)
+"""
+
+toolchain = {'name': 'GCC', 'version': '12.3.0'}
+
+# source_urls = ['https://github.com/vibansal/HapCUT2/archive/refs/tags/']
+# sources = ['%(version)s.tar.gz']
+
+download_instructions = """
+  The last version tag v1.3.3 does not include last modification that make it work.
+  So we download source from master branch at the commit where I tested.
+"""
```

Comment:
`download_instructions` are only meant for sources that are not downloaded by EB but have to be downloaded manually before running EB.
can you change this to a comment?